### ISSUE TITLE
[yast2-snapper] 'Refresh list' button on Snapper's snapshots window

### ISSUE THREAD

**Issue by MikhailKasimov** [yast/yast-yast2#713]
Hello!

Not sure, if it is right place to put this report (here or to Snapper repo), but anyway, there is one use-case, that can clearly show, that ```yast2-spanner``` window should have 'Refresh list' button to give to user possibility to update snapshots list without closing the Snapper's snapshots window. Because it's very inconvenient to close->open->close->open Snapper window to update snapshots list. Thanks!

Long story short: https://youtu.be/ZdaWekuMlqI

Please, consider! Thanks!

**Comment by aschnell** [comment]
Thanks for the enhancement request. Implemented in https://github.com/yast/yast-snapper/pull/66,
